subsys/bluetooth/controller/hci/hci.c

```diff
@@ -103,6 +103,55 @@ static uint64_t event_mask = DEFAULT_EVENT_MASK;
 static uint64_t event_mask_page_2 = DEFAULT_EVENT_MASK_PAGE_2;
 static uint64_t le_event_mask = DEFAULT_LE_EVENT_MASK;
 
+static struct net_buf *cmd_complete_status(uint8_t status);
+
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+#if defined(CONFIG_BT_HCI_RAW)
+static int adv_cmds_legacy_check(struct net_buf **cc_evt)
```

Comment:
Wouldn't it be nicer to have a single set of functions and then have the `#if defined(CONFIG_BT_HCI_RAW)` checks inside the functions?

Reply:
indeed, will fix